src/main/java/io/aiven/kafka/connect/http/HttpSinkTask.java

```diff
@@ -53,23 +59,61 @@ public void start(final Map<String, String> props) {
         if (this.httpSender == null) {
             this.httpSender = HttpSender.createHttpSender(config);
         }
+
         recordSender = RecordSender.createRecordSender(httpSender, config);
+
+        this.useLegacySend = config.batchingEnabled();
+
         if (Objects.nonNull(config.kafkaRetryBackoffMs())) {
             context.timeout(config.kafkaRetryBackoffMs());
         }
+
+        try {
+            if (context.errantRecordReporter() == null) {
+                log.info("Errant record reporter not configured.");
+            }
+
+            // may be null if DLQ not enabled
+            reporter = context.errantRecordReporter();
+        } catch (NoClassDefFoundError | NoSuchMethodError e) {
+            // Will occur in Connect runtimes earlier than 2.6
+            log.warn("AK versions prior to 2.6 do not support the errant record reporter.");
```

Comment:
Nitpick: I would prefer to rather use Apache Kafka here not AK

Reply:
Done. No worries on nitpicks - I am quite fine with them.